aatp/peerbook_webrtc/admin.spec.ts

```diff
@@ -361,6 +361,7 @@ test.describe('peerbook administration', ()  => {
         await page.keyboard.press("Enter")
         await sleep(100)
         await page.keyboard.press("ArrowDown")
+        await page.evaluate(() => { terminal7.log('test') })
```

Comment:
`test` is not good enough because you already have `test issue` as the description. Please change to `log line` and add a test at the end of the function to ensure it's in the body.

Reply:
runner-1      |     Expected pattern: /test issue/
runner-1      |     Received string:  "--89132905806bb7a3ef1c2fe0eefd353389188eada99c1b8903bee1a25628·
runner-1      |     Content-Transfer-Encoding: 8bit·
runner-1      |     Content-Type: text/plain; charset=UTF-8···
runner-1      |     Click & verify your new Terminal7 client·····
runner-1      |     Dear User,···
runner-1      |     Seems like there's a Terminal7 client that is waiting your approval.·
runner-1      |     To approve please click - http://peerbook:17777/verify/sjUBLNkB5uvt7ZO2o9qZFCJTwh8SENFF8_b6d0i2···
runner-1      |     If you find this email wrong, we're sorry. Please reply to let us know what is wrong.···
runner-1      |     PeerBook emailer·
runner-1      |     http://peerbook:17777···
runner-1      |     --89132905806bb7a3ef1c2fe0eefd353389188eada99c1b8903bee1a25628·
runner-1      |     Content-Transfer-Encoding: 8bit·
runner-1      |     Content-Type: text/html; charset=UTF-8···
runner-1      |     <title>Click & verify your new Terminal7 client</title>·
runner-1      |     </head>·
runner-1      |     <p>·
runner-1      |     Dear User,·
runner-1      |     </p>·
runner-1      |     <p>·
runner-1      |     Seems like there's a Terminal7 client that is waiting your approvel.·
runner-1      |     Please click <a href=\"http://peerbook:17777/verify/sjUBLNkB5uvt7ZO2o9qZFCJTwh8SENFF8_b6d0i2\">here</a> to approve.·
runner-1      |     </p>·
runner-1      |     <p>·

im getting this error 


and this one: 
runner-1      |     If you find this email wrong, we're sorry. Please reply·
runner-1      |     to let us know what is wrong.·
runner-1      |     </p>·
runner-1      |     <br>·
runner-1      |     PeerBook emailer·
runner-1      |     <br>·
runner-1      |     http://peerbook:17777···
runner-1      |     --89132905806bb7a3ef1c2fe0eefd353389188eada99c1b8903bee1a25628--"
runner-1      | 
runner-1      |       374 |         expect(msg.count).toBe(1)
runner-1      |       375 |         const body = msg.items[0].Content.Body
runner-1      |     > 376 |         expect(body).toMatch(/test issue/)
runner-1      |           |                      ^
runner-1      |       377 |         expect(body).toMatch(/log line/)
runner-1      |       378 |     })
runner-1      |       379 | })

i think you need to change the path so i can check it properly